### PR TITLE
Disabled the blue outline on disabled text inputs to signify it is inactive

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
Regarding #1462

### CODE REVIEW

**Comment by iwehrman** [comment]
Did you figure out how to get a disabled text input??

**Comment by DivyaPrabhakar** [comment]
@iwehrman Yup!

Create document
Places some shapes on it 
Deselect everything -- The W and H inputs are disabled. 

I think it also works if you:
Create new document
Create Artboard 
Have nothing selected -- all the inputs are inactive

File: src/style/shared/number-input.less [hunk 1]
```diff
@@ -30,6 +30,10 @@
     border: @hairline solid @highlight;
 }
 
+.number-input:disabled:hover {
+	border: none;
```
Comment (iwehrman) [inline src/style/shared/number-input.less]: This is a tab character. Thou shalt not use tab characters! Instead, you must configure your editor to _always_ create four-space tab stops.

Comment (iwehrman) [inline src/style/shared/number-input.less]: Looks good otherwise :)

**Pull request closed without merging**